src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -179,8 +190,12 @@ void CARMACloudPlugin::CARMAResponseHandler(QHttpEngine::Socket *socket)
 	
 	string tcm = _cloudUpdate;
 
-	cout<<"Received this from cloud"<<tcm<<endl;
-
+	PLOG(logINFO) << "Received this from cloud" << tcm << std::endl;

```

Comment:
A more descriptive log statement would be better. Maybe something like "TCM received from carma-cloud" or something indicating what was received.

Reply:
Actually just saw a different log statement for "Sent TCR : ". We should probably make these log statements look similar.  So maybe "Received TCM : "